src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/ArtworkAuctionCreateAlertHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, Text } from "@artsy/palette"
+import * as React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtworkAuctionCreateAlertHeader_artwork$data } from "__generated__/ArtworkAuctionCreateAlertHeader_artwork.graphql"
+import { ArtworkSidebarCreateAlertButtonFragmentContainer } from "Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCreateAlertButton"
+import { useFeatureFlag } from "System/useFeatureFlag"
+
+interface ArtworkAuctionCreateAlertHeaderProps {
+  artwork: ArtworkAuctionCreateAlertHeader_artwork$data
+}
+
+const ArtworkAuctionCreateAlertHeader: React.FC<ArtworkAuctionCreateAlertHeaderProps> = ({
+  artwork,
+}) => {
+  const getArtistNames = () => {
+    if (!artwork.artistNames) {
+      return "Artist Unavailable"
+    }
+    return artwork.artistNames
+  }
+
+  const isArtworkAuctionCreateAlertHeader = useFeatureFlag(
+    "onyx_auction-header-alert-cta"
+  )
```

Comment:
**suggestion:** we usually add "Enabled" at the end of such variables `isArtworkAuctionCreateAlertHeaderEnabled` or `isAuctionResultCreateAlertHeaderEnabled`